pkg/pool-weighted/test/foundry/LiquidityApproximationWeighted.t.sol

```diff
@@ -104,7 +110,7 @@ contract LiquidityApproximationWeightedTest is LiquidityApproximationTest {
     ) public {
         // Weights can introduce some differences in the swap fees calculated by the pool during unbalanced add/remove
         // liquidity, so the error tolerance needs to be a bit higher than the default tolerance.
-        roundingDelta = 1e13;
+        defectRoundingDelta = 0.00001e16; // 0.00001%
```

Comment:
These specific values were all determined empirically?

Reply:
Yes. I was a bit on the fence on this one, but it gives some insights to have them differentiated.
The test verifies that the defect is absorbed by the fees anyways, which is why I bumped the minimum swap fee for the pools.

This is just a first rough approximation though. In the next iteration of the tests we'll be measuring invariants before and after the operations to ensure that they are not going down in any case, which is a stronger assurance (even for 0 fees).